web/src/util/roundedPercents.test.js

```diff
@@ -0,0 +1,14 @@
+import roundedPercents from './roundedPercents';
+
+describe('rounding percentages util', () => {
+  test('spreads a number across a set of percentages', () => {
+    expect(roundedPercents(224, [0.15, 0.19, 0.3, 0.36])).toEqual([
```

Comment:
Picked these values because if you split 224 into 15%, 19%, 30%, and 36%, round all those pieces up, and then add them together, you get 225.  So...  It's useful to know our magic thingy doesn't do that.